src/languages/lib/swift.js

```diff
@@ -0,0 +1,285 @@
+  'dump',
+  'fatalError',
+  'getVaList',
+  'isKnownUniquelyReferenced',
+  'max',
+  'min',
+  'numericCast',
+  'pointwiseMax',
+  'pointwiseMin',
+  'precondition',
+  'preconditionFailure',
+  'print',
+  'readLine',
+  'repeatElement',
+  'sequence',
+  'stride',
+  'swap',
+  'swift_unboxFromSwiftValueWithType',
+  'transcode',
+  'type',
+  'unsafeBitCast',
+  'unsafeDowncast',
+  'withExtendedLifetime',
+  'withUnsafeMutablePointer',
+  'withUnsafePointer',
+  'withVaList',
+  'withoutActuallyEscaping',
+  'zip',
+];
+
+// Valid first characters for operators.
+export const operatorHead = either(
+  /[/=\-+!*%<>&|^~?]/,
+  /[\u00A1–\u00A7]/,
+  /[\u00A9\u00AB]/,
+  /[\u00AC\u00AE]/,
+  /[\u00B0\u00B1]/,
+  /[\u00B6\u00BB\u00BF\u00D7\u00F7]/,
+  /[\u2016–\u2017]/,
+  /[\u2020–\u2027]/,
+  /[\u2030–\u203E]/,
+  /[\u2041–\u2053]/,
+  /[\u2055–\u205E]/,
+  /[\u2190–\u23FF]/,
+  /[\u2500–\u2775]/,
+  /[\u2794–\u2BFF]/,
+  /[\u2E00–\u2E7F]/,
+  /[\u3001–\u3003]/,
+  /[\u3008–\u3020]/,
+  /[\u3030]/
+);
+
+// Valid characters for operators.
+export const operatorCharacter = either(
+  operatorHead,
+  /[\u0300–\u036F]/,
+  /[\u1DC0–\u1DFF]/,
+  /[\u20D0–\u20FF]/,
+  /[\uFE00–\uFE0F]/,
+  /[\uFE20–\uFE2F]/,
+  // TODO: The following characters are also allowed, but the regex isn't supported yet.
+  // /[\u{E0100}–\u{E01EF}]/u
+);
+
+// Valid first characters for identifiers.
+export const identifierHead = either(
+  /[a-zA-Z_]/,
+  /[\u00A8\u00AA\u00AD\u00AF\u00B2–\u00B5\u00B7–\u00BA]/,
+  /[\u00BC–\u00BE\u00C0–\u00D6\u00D8–\u00F6\u00F8–\u00FF]/,
+  /[\u0100–\u02FF\u0370–\u167F\u1681–\u180D\u180F–\u1DBF]/,
+  /[\u1E00–\u1FFF]/,
+  /[\u200B–\u200D\u202A–\u202E\u203F–\u2040\u2054\u2060–\u206F]/,
+  /[\u2070–\u20CF\u2100–\u218F\u2460–\u24FF\u2776–\u2793]/,
+  /[\u2C00–\u2DFF\u2E80–\u2FFF]/,
+  /[\u3004–\u3007\u3021–\u302F\u3031–\u303F\u3040–\uD7FF]/,
+  /[\uF900–\uFD3D\uFD40–\uFDCF\uFDF0–\uFE1F\uFE30–\uFE44]/,
+  /[\uFE47–\uFFFD]/,
+  // The following characters are also allowed, but the regexes aren't supported yet.
+  // /[\u{10000}–\u{1FFFD}\u{20000–\u{2FFFD}\u{30000}–\u{3FFFD}\u{40000}–\u{4FFFD}]/u,
+  // /[\u{50000}–\u{5FFFD}\u{60000–\u{6FFFD}\u{70000}–\u{7FFFD}\u{80000}–\u{8FFFD}]/u,
+  // /[\u{90000}–\u{9FFFD}\u{A0000–\u{AFFFD}\u{B0000}–\u{BFFFD}\u{C0000}–\u{CFFFD}]/u,
+  // /[\u{D0000}–\u{DFFFD}\u{E0000–\u{EFFFD}]/u
+);
+
+// Valid characters for identifiers.
+export const identifierCharacter = either(
+  identifierHead,
+  /\d/,
+  /[\u0300–\u036F\u1DC0–\u1DFF\u20D0–\u20FF\uFE20–\uFE2F]/
+);
+
+// Valid identifier.
+export const identifier = concat(identifierHead, identifierCharacter, '*');
+
+// Built-in attributes, which are highlighted as keywords.
+// @available is handled separately.
+export const keywordAttributes = [
+  /autoclosure/,
+  concat(/convention\(/, either('swift', 'block', 'c'), /\)/),
+  /discardableResult/,
+  /dynamicCallable/,
+  /dynamicMemberLookup/,
+  /escaping/,
+  /frozen/,
+  /GKInspectable/,
```

Comment:
If they (in the context of the list) are literal strings, then prefer normal quoting not `//`.

Reply:
Can I mix strings and regexes in the same array, will that work?

Reply:
Done.

Reply:
Mixing isn't a problem.  The underlying utility stuff and compiler usually handles strings and regex interchangeably without any issue.  It's only an issue when doing something like `a+b` or string concatenation, which mostly we're trying to get away from.